2021-10-16 17:04:26,246 INFO    MainThread:49648 [wandb_setup.py:_flush():69] setting env: {}
2021-10-16 17:04:26,246 INFO    MainThread:49648 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-16 17:04:26,246 INFO    MainThread:49648 [wandb_setup.py:_flush():69] setting login settings: {'api_key': 'd909c6dbfbe16759d28353a2732b1a859c44fb4c'}
2021-10-16 17:04:26,246 INFO    MainThread:49648 [wandb_init.py:_log_setup():357] Logging user logs to C:\Users\nongaya\Desktop\DE_Week\Day3\Project_Introduction to Regression\wandb\run-20211016_170426-3u68fmun\logs\debug.log
2021-10-16 17:04:26,246 INFO    MainThread:49648 [wandb_init.py:_log_setup():358] Logging internal logs to C:\Users\nongaya\Desktop\DE_Week\Day3\Project_Introduction to Regression\wandb\run-20211016_170426-3u68fmun\logs\debug-internal.log
2021-10-16 17:04:26,257 INFO    MainThread:49648 [wandb_init.py:_jupyter_setup():307] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001A75AC84490>
2021-10-16 17:04:26,257 INFO    MainThread:49648 [wandb_init.py:init():390] calling init triggers
2021-10-16 17:04:26,257 INFO    MainThread:49648 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-16 17:04:26,257 INFO    MainThread:49648 [wandb_init.py:init():435] starting backend
2021-10-16 17:04:26,257 INFO    MainThread:49648 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-10-16 17:04:26,257 INFO    MainThread:49648 [backend.py:ensure_launched():135] starting backend process...
2021-10-16 17:04:26,396 INFO    MainThread:49648 [backend.py:ensure_launched():139] started backend process with pid: 70400
2021-10-16 17:04:26,396 INFO    MainThread:49648 [wandb_init.py:init():440] backend started and connected
2021-10-16 17:04:26,406 INFO    MainThread:49648 [wandb_run.py:_label_probe_notebook():833] probe notebook
2021-10-16 17:04:26,416 INFO    MainThread:49648 [wandb_run.py:_label_probe_notebook():843] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-16 17:04:26,416 INFO    MainThread:49648 [wandb_init.py:init():499] updated telemetry
2021-10-16 17:04:26,416 INFO    MainThread:49648 [wandb_init.py:init():522] communicating current version
2021-10-16 17:04:31,127 INFO    MainThread:49648 [wandb_init.py:init():527] got version response 
2021-10-16 17:04:31,127 INFO    MainThread:49648 [wandb_init.py:init():535] communicating run to backend with 30 second timeout
2021-10-16 17:04:31,648 INFO    MainThread:49648 [wandb_init.py:init():562] starting run threads in backend
2021-10-16 17:04:36,656 INFO    MainThread:49648 [wandb_run.py:_console_start():1651] atexit reg
2021-10-16 17:04:36,657 INFO    MainThread:49648 [wandb_run.py:_redirect():1525] redirect: SettingsConsole.WRAP
2021-10-16 17:04:36,657 INFO    MainThread:49648 [wandb_run.py:_redirect():1562] Wrapping output streams.
2021-10-16 17:04:36,658 INFO    MainThread:49648 [wandb_run.py:_redirect():1586] Redirects installed.
2021-10-16 17:04:36,659 INFO    MainThread:49648 [wandb_init.py:init():587] run started, returning control to user process
2021-10-16 17:04:37,085 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:04:37,085 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:06:44,401 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:06:44,401 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:06:44,401 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:08:43,768 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:08:43,768 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:08:43,768 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:08:58,171 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:08:58,173 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:08:58,173 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:09:04,372 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:09:04,372 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:09:04,373 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:09:18,160 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:09:18,161 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:09:18,161 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:10:03,510 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:10:03,512 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:10:03,513 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:10:15,375 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:10:15,834 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:10:15,834 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:11:19,806 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:11:19,807 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:11:19,807 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:14:25,676 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:14:25,677 INFO    MainThread:49648 [wandb_config.py:__setitem__():143] config set tol = 0.01 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001A762A35E50>>
2021-10-16 17:14:25,677 INFO    MainThread:49648 [wandb_run.py:_config_callback():878] config_cb tol 0.01 None
2021-10-16 17:14:25,677 INFO    MainThread:49648 [wandb_config.py:__setitem__():143] config set penalty = 12 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001A762A35E50>>
2021-10-16 17:14:25,677 INFO    MainThread:49648 [wandb_run.py:_config_callback():878] config_cb penalty 12 None
2021-10-16 17:14:25,678 INFO    MainThread:49648 [wandb_config.py:__setitem__():143] config set C = 1.0 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000001A762A35E50>>
2021-10-16 17:14:25,678 INFO    MainThread:49648 [wandb_run.py:_config_callback():878] config_cb C 1.0 None
2021-10-16 17:14:25,678 INFO    MainThread:49648 [wandb_init.py:_pause_backend():270] pausing backend
2021-10-16 17:14:25,679 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:14:44,284 INFO    MainThread:49648 [wandb_init.py:_resume_backend():279] resuming backend
2021-10-16 17:14:44,295 INFO    MainThread:49648 [wandb_setup.py:_flush():69] setting env: {}
2021-10-16 17:14:44,295 INFO    MainThread:49648 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-16 17:14:44,295 INFO    MainThread:49648 [wandb_setup.py:_flush():69] setting login settings: {'api_key': 'd909c6dbfbe16759d28353a2732b1a859c44fb4c'}
2021-10-16 17:14:44,295 INFO    MainThread:49648 [wandb_init.py:_log_setup():357] Logging user logs to C:\Users\nongaya\Desktop\DE_Week\Day3\Project_Introduction to Regression\wandb\run-20211016_171444-m9dkqd4b\logs\debug.log
2021-10-16 17:14:44,296 INFO    MainThread:49648 [wandb_init.py:_log_setup():358] Logging internal logs to C:\Users\nongaya\Desktop\DE_Week\Day3\Project_Introduction to Regression\wandb\run-20211016_171444-m9dkqd4b\logs\debug-internal.log
2021-10-16 17:14:44,298 INFO    MainThread:49648 [wandb_init.py:init():390] calling init triggers
2021-10-16 17:14:44,298 INFO    MainThread:49648 [wandb_init.py:init():395] wandb.init called with sweep_config: {}
config: {}
2021-10-16 17:14:44,298 INFO    MainThread:49648 [wandb_init.py:init():410] re-initializing run, found existing run on stack: 3u68fmun
2021-10-16 17:14:44,299 INFO    MainThread:49648 [wandb_run.py:finish():1311] finishing run python2020-/Mobile Recommendation_Plan/3u68fmun
2021-10-16 17:14:44,300 INFO    MainThread:49648 [jupyter.py:save_history():428] not saving jupyter history
2021-10-16 17:14:44,300 INFO    MainThread:49648 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2021-10-16 17:14:44,300 INFO    MainThread:49648 [wandb_init.py:_jupyter_teardown():289] cleaning up jupyter logic
2021-10-16 17:14:44,300 INFO    MainThread:49648 [wandb_run.py:_atexit_cleanup():1621] got exitcode: 0
2021-10-16 17:14:44,301 INFO    MainThread:49648 [wandb_run.py:_restore():1593] restore
2021-10-16 17:14:44,559 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 838
}

2021-10-16 17:14:44,849 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 838
}

2021-10-16 17:14:45,379 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 838
}

2021-10-16 17:14:45,833 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39301
}

2021-10-16 17:14:45,934 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,038 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,142 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,243 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,347 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,449 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,550 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 838
  total_bytes: 39499
}

2021-10-16 17:14:46,652 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:46,756 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:46,857 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:46,959 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:47,075 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:47,184 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:47,295 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:47,405 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:49,098 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}

2021-10-16 17:14:50,112 INFO    MainThread:49648 [wandb_run.py:_wait_for_finish():1746] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 39499
  total_bytes: 39499
}
local_info {
}

2021-10-16 17:14:51,244 INFO    MainThread:49648 [wandb_run.py:_append_history():1959] rendering history
2021-10-16 17:14:51,245 INFO    MainThread:49648 [wandb_run.py:_append_summary():1914] rendering summary
2021-10-16 17:14:51,245 INFO    MainThread:49648 [wandb_run.py:_append_files():2009] logging synced files
